extensions/ql-vscode/package.json

```diff
@@ -644,7 +644,7 @@
       },
       {
         "command": "codeQLQueryHistory.openOnGithub",
-        "title": "View Logs"
+        "title": "Open on GitHub"
```

Comment:
I don't necessarily disagree with this change, but it's worth noting that this will be a user-visible change because it's visible in the query history context menu. Is there a desire to change the name of this menu option?

Reply:
Good point, I did this because this is one of the only commands where the command name doesn't match the title, but we'll probably want to keep it as-is. I'll revert the change.